invenio_config_tugraz/permissions/policies.py

```diff
@@ -311,3 +333,174 @@ class TUGrazRDMRecordPermissionPolicy(RecordPermissionPolicy):
     # Used to hide the `parent.is_verified` field. It should be set to
     # correct permissions based on which the field will be exposed only to moderators
     can_moderate = [SystemProcess()]
+
+
+class TUGrazCommunityPermissionPolicy(BasePermissionPolicy):
```

Comment:
why do you use `BasePermissionPolicy` from `invenio_communities.permissions`?

Reply:
for context, `BasePermissionPolicy` is originally from `invenio_records_permissions`

however, this import should trace the base class of `CommunityPermissionPolicy`
importing it from invenio-communities better guarantees that

I'll add a comment at import-site to clarify...